.github/workflows/e2e.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
             --create-namespace \
             --atomic
       - name: Install sealed-secrets-web with yaml format
-        run: ./testdata/e2e/installSealedSecretsWebChart.sh yaml ""
+        run: ./testdata/e2e/installSealedSecretsWebChart.sh yaml "" "sealed-secrets" "sealed-secrets"
```

Comment:
This is already the default https://github.com/bakito/sealed-secrets-web/blob/main/chart/values.yaml#L19

Reply:
true, will revert